css/main.css

```diff
@@ -560,3 +561,69 @@ textarea {
     height: 32px;
   }
 }
+
+/* hamburger activation */
+.mobile-links {
+  display: none;
```

Comment:
Kindly fix the hamburger menu. When you open it it's not showing properly the menu. It doesn't match the Figma template.
![Screenshot from 2022-09-15 21-08-19](https://user-images.githubusercontent.com/98363075/190541066-9fe2a8f6-bb0b-4729-b8f4-eb3e918b66e2.png) ![Screenshot from 2022-09-15 21-09-01](https://user-images.githubusercontent.com/98363075/190541158-7a8050ce-94aa-4fe7-bef3-27dbc5793740.png)